package.json

```diff
@@ -27,6 +27,7 @@
     "@ngrx/store-devtools": "^8.0.1",
     "hammerjs": "^2.0.8",
     "ngrx-store-freeze": "^0.2.4",
+    "ngx-mat-select-search": "^1.7.6",
```

Comment:
Need this package since Angular Material doesn't have a built-in way to filter a dropdown.